### PR TITLE
Tranlated to Turkish completely

### PR DESCRIPTION
**Pull request by ahmetlii**
Yay!

### CODE REVIEW

**Review by jacob-g — APPROVED** [review]
Since I don't know Turkish, I'm operating under the assumption that the translations are correct.

File: i18n/tr.json [hunk 1]
```diff
@@ -94,11 +99,19 @@
...
+
+ "scratch-confirmaccount-request-account-login-notice": "Bir hesap istiyorsanız, bir tane [[Special:RequestAccount|bir tane istemelisiniz]].",
+ "scratch-confirmaccount-view-request": "Halihazırda istendi mi? [[Special:RequestAccount/FindRequest|İsteğinizi görün]].",
+ "scratch-confirmaccount-dropdown-other": "Diğer",
+ "scratch-confirmaccount-common-admin-comments": "* Kabul nedenleri\n** İsteğiniz kabul edildi! Özel:KullanıcıOturumuAçma sayfasından seçtiğiniz kullanıcı adıyla ve şifrenizle giriş yapabilirsiniz.\n** Yorum için teşekkürler. İsteğinizi onayladım. Özel:KullanıcıOturumuAçma sayfasından seçtiğiniz kullanıcı adıyla ve şifrenizle giriş yapabilirsiniz.\n* Ret nedenleri\n** İstek listesindeki gereksinimleri karşılamadığınızdan ötürü isteğinizi reddettim. Gereksinimleri sağladığınız zaman yeniden hesap isteyebilirsiniz.\n** İstek formunu düzgün bir şekilde doldurmadığınız için isteğinizi reddettim.\n* Yorumlar\n** ___ diyerek ne demek istediğinizi açıklığa kavuşturabilir misiniz?\n** İsteğe daha fazla detay ekleyebilir misiniz?\n** ___ için daha fazla örnek ekleyebilir misiniz lütfen?"
```
Comment (Kenny2github) [inline i18n/tr.json:116]: why no indent?

File: i18n/tr.json [hunk 2]
```diff
@@ -94,11 +99,19 @@
 	"scratch-confirmaccount-email-resent": "Doğrulama e-postası yeniden gönderildi.",
 	"scratch-confirmaccount-new-scratcher": "Yeni Scratch'çiler hesap isteği gönderemez.",
 	"scratch-confirmaccount-joinedat": "Bir hesap istemek için en az $1 gündür Scratch'te kayıtlı olmanız gerekir.",
+	"scratch-confirmaccount-profile-error": "Hesabınızı kontrol ederken bir hata gerçekleşti. Daha sonra tekrar deneyin.",
 
 	"scratch-confirmaccount-requests-awaiting": "<b>$2</b> $1 $3 bekliyor. <b>İlginiz gerekli!</b>",
-	"scratch-confirmaccount-requests-awaiting-linktext": "{{PLURAL:$1|istek|istek}}",
-	"scratch-confirmaccount-requests-awaiting-isare": "{{PLURAL:$1|bekliyor|bekliyor}}",
+	"scratch-confirmaccount-requests-awaiting-linktext": "istek",
```
Comment (Kenny2github) [inline i18n/tr.json:105]: The correct way to do this is `{{PLURAL:$1|istek}}` - even if there's no other form.

**Review by Kenny2github — CHANGES_REQUESTED** [review]

File: i18n/tr.json (hunk 2 above)
Comment (ahmetlii) [reply i18n/tr.json:105]: ok

File: i18n/tr.json (hunk 1 above)
Comment (ahmetlii) [reply i18n/tr.json:116]: i forgot to add

**Review by Kenny2github — APPROVED** [review]

**Pull request merged**